server/webapp/WEB-INF/rails/app/presenters/api_v1/shared/config_origin/config_repo_summary_representer.rb

```diff
@@ -20,16 +20,22 @@ module ConfigOrigin
       class ConfigRepoSummaryRepresenter < BaseRepresenter
         alias_method :config_repo, :represented
 
+        def urlFor(url_builder, path)
```

Comment:
I see this block of code has been repeated at many places, ([Pipeline Representer](https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails/app/presenters/api_v1/dashboard/pipeline_representer.rb#L35-L55), [Pipeline Config Summary Representer](https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails/app/presenters/api_v2/config/pipeline_config_summary_representer.rb#L23-L35)) can this method be pushed under `BaseRepresenter`? so that we can reuse it rather than defining at each representer?

Reply:
That seems like a good improvement. I’ll take care and if it